delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/ProjectsDummy.scala

```diff
@@ -130,6 +134,18 @@ final class ProjectsDummy private (
 
   override def currentEvents(offset: Offset): fs2.Stream[Task, Envelope[ProjectEvent]] =
     journal.currentEvents(offset)
+
+  override def provisionProject(subject: Subject): UIO[Unit] = subject match {
+    case user @ User(subject, realm) if provisioningConfig.enabled =>
+      (for {
+        org       <- IO.fromOption(provisioningConfig.enabledReams.get(realm))
+        proj      <- IO.fromEither(Label.apply(subject)).mapError { _ => () }
+        projectRef = ProjectRef(org, proj)
+        exists    <- cache.fetch(projectRef).map(_.isDefined)
```

Comment:
should we use the cache, or the. regular `fetch` ?

Reply:
I think cache should be faster here.